msal/src/Microsoft.Identity.Client/ClientApplicationBase.cs

```diff
@@ -148,23 +179,33 @@ public async Task<IEnumerable<IAccount>> GetAccountsAsync()
         }
 
         /// <summary>
-        /// Get the account by identifier from the accounts available in the cache.
+        /// Get the <see cref="IAccount"/> by its identifier among the accounts available in the token cache.
         /// </summary>
-        /// <param name="accountId">account identifier</param>
+        /// <param name="accountId">account identifier. The value of the identifier will probably have been stored value from the
```

Comment:
>The value of the identifier will probably have been stored value [](start = 56, length = 64)

imo: hard to understand. Suggestion:
The identifier is typically <se cref= ..

Reply:
Good catch!